templates/tfengine/components/resources/compute_networks/main.tf

```diff
@@ -29,7 +29,11 @@ module "{{$resource_name}}" {
     {
```

Comment:
Judging from the build failure, I think the `{{if has . "subnets" -}}` needs an else clause that sets subnets to an empty list.

Reply:
Yeah, I just set a subnet and made them required, since it doesn't seem like there should be networks without subnets.